eve/main.yml

```diff
@@ -305,6 +305,7 @@ stages:
         env:
           ENABLE_UTAPI_V2: t
           S3BACKEND: mem
+          BUCKET_DENY_FILTER: utapi-event-filter-deny-bucket
```

Comment:
I'd suggest having two buckets in this list and check each of them in turn in the tests, to cover parsing of the env var for multiple buckets

Reply:
```suggestion
          UTAPI_BUCKET_DENY_FILTER: utapi-event-filter-deny-bucket
```

Reply:
I don't think it will support multiple buckets without some more sed magic. This was only added for test setup, so maybe somewhere else would be easier to add multiple buckets. I'll see what I can do

Reply:
Ah, I though we were setting this variable in the same way than in the production environment, if it's just a test helper, then no need to change it since it would not increase the test coverage.